app/models/user.rb

```diff
@@ -38,4 +38,13 @@ class User < ApplicationRecord
   def full_name
     "#{first_name} #{last_name}"
   end
+
+  def update_with_password(params = {})
```

Comment:
Two notes:

1. You might wanted to write `update_withtout_password` no?
2. You should unit test this method, in the `spec/models/user_spec.rb`. ;)